client/next.config.js

```diff
@@ -2,6 +2,7 @@
 const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
+  output: 'standalone',
   publicRuntimeConfig: {
     API_URL: "http://localhost:8080",
```

Comment:
I am not sure how this is read from the NextJS. Does it override the `.env`?

Reply:
the `nextConfig` does not override environment variables, but can be set with them. All envars in next are exposed through Node's `process.env` object, and can be used to set values here. These are typically used by the next.js build system, but you can access his programmatically in code using the frameworks exported `getConfig` function. It returns `any` so you will need to cast the object to reference it via Typescript.

Reply:
thanks, now we're not using `NEXT_PUBLIC` env vars anywhere, only `publicRuntimeConfig` object